<launch>
	  <arg name="manual_routing" default="True"/>

		<node name="mission_coordinator_client" pkg="nautonomous_mission_coordinator" type="mission_coordinator_client.py" output="screen">
			<param name="manual_routing" value="$(arg manual_routing)"/>
		</node>

</launch>
